### ISSUE TITLE
puppet lint default does not check autoload class layout

### ISSUE THREAD

**Issue by abuxton** [rodjek/puppet-lint#805]
seems autoload layout is not actually checked for in default checks. 
http://puppet-lint.com/checks/autoloader_layout/
https://puppet.com/docs/puppet/5.3/style_guide.html#separate-files

checking with PDK, on an example class
``` puppet
class amoduleclass
}
class anotherclass{
}
define defineanotherclass(
){
}
```
this validates correctly, when it should not.

**Comment by abuxton** [comment]
Ignore invalid test, false negative
